spec/support/request_helpers.rb

```diff
@@ -47,8 +47,6 @@ def verify_merchant_attributes(merchant, db_merchant)
     expect(merchant).to have_key('name')
     expect(merchant['name']).to eq(db_merchant.name)
     expect(merchant['name']).to be_a(String)
-
-    verify_timestamps(merchant, db_merchant) 
```

Comment:
I assume because we do not have the timestamps in the serializer, we no longer need to verify them? @tmikeschu